# sensory data (i.e., the response of table top object detection)
sensor_msgs/PointCloud2 cloud_plane
float32[4] cloud_plane_coef
sensor_msgs/PointCloud2[] objects
---
int32 detected_touch_index
bool success
